docs/reference/alert-schema.asciidoc

```diff
@@ -112,7 +112,12 @@ Type: keyword
 |`signal.rule.type` |`kibana.alert.rule.type`| Type: keyword
 |`signal.rule.updated_at` |`kibana.alert.rule.updated_at`| Type: date
 |`signal.rule.updated_by` |`kibana.alert.rule.updated_by`| Type: keyword
-|`signal.rule.version` |`kibana.alert.rule.version`| Type: keyword
+|`signal.rule.version` |`kibana.alert.rule.version`| A number corresponding to the version of a rule distributed by Elastic.
```

Comment:
Super small nit: is a "rule distributed by Elastic" a prebuilt rule? If it is, do we want to mention that in the description since users might be familiar with them? For example:
```suggestion
|`signal.rule.version` |`kibana.alert.rule.version`| A number corresponding the version of a prebuilt rule distributed by Elastic.
```